docs/DeveloperGuide.md

```diff
@@ -854,6 +854,46 @@ Prerequisite: Only 1 guest to be edited. The guest's index should exist.
    3. Other incorrect edit commands to try: `edit`, `edit x`, `edit 1 rc/hi`, `edit ...`, `edit 1 n/Jo@`<br>
       Expected: No guest is edited, because the **parameters are invalid**. Error details shown in the result display.
 
+   5. Other incorrect edit commands to try: `edit`, `edit x`, `edit 1 rc/hi`, `edit ...`<br>
+      Expected: No guest is edited, because the parameters are invalid. Error details shown in the status message.
+      Status bar remains the same.
+
+### Marking Guests Rooms as Unclean
+
+Prerequisite: All guests room cleaning status will be edited.
+
+   1. Test case: `markroomsunclean`<br>
+      Expected: All guests are marked as unclean, the icon representing `isRoomClean` will change to red from green.
+   2. Test case: `markroomunclean`<br>
+      Expected: The error will throw: "unknown command", because the command is **missing out the `s`**
+   3. Test case: `markRoomsUnclean`<br>
+      Expected: The error will throw: "unknown command", because the command is **Capitalising the `R` or `U`**
+
+### Changing guests bill value
+
+Prerequisite: Only one guest's bill will be edited.
+The guest should exist in GuestBook. The format and content of the command should be valid.
+The bill value cannot exceed 999,999,999,999.99.
```

Comment:
Can we add that "The test cases must be executed in succession as state below for the intended outcome".